components/elm/src/biogeophys/SurfaceAlbedoMod.F90

```diff
@@ -235,17 +241,38 @@ subroutine SurfaceAlbedo(bounds,     &
 
     ! Cosine solar zenith angle for next time step
 
+    deg2rad = SHR_CONST_PI/180._r8
     do g = bounds%begg,bounds%endg
```

Comment:
I believe it would be better to refactor the code such that the `if (use_finetop_rad)` is outside the do-loop as:

```fortran
if (.not. use_finetop_rad) then

   do g = bounds%begg,bounds%endg
       coszen_gcell(g) = shr_orb_cosz (nextsw_cday, grc_pp%lat(g), grc_pp%lon(g), declinp1)
       cosinc_gcell(g) = coszen_gcell(g)
   enddo

else

   do g = bounds%begg,bounds%endg
       coszen_gcell(g) = shr_orb_cosz (nextsw_cday, grc_pp%lat(g), grc_pp%lon(g), declinp1)
       ...
   enddo

enddo
```